packages/bigcommerce-payments-integration/src/BigCommercePaymentsPaypal/BigCommercePaymentsPaymentMethod.test.tsx

```diff
@@ -0,0 +1,158 @@
+import { AccountInstrument, createCheckoutService } from '@bigcommerce/checkout-sdk';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { Formik } from 'formik';
+import { noop } from 'lodash';
+import React, { FunctionComponent } from 'react';
+
+import { createLocaleContext, LocaleContext } from '@bigcommerce/checkout/locale';
+import {
+    CheckoutContext,
+    PaymentFormContext,
+    PaymentMethodProps,
+} from '@bigcommerce/checkout/payment-integration-api';
+import { getPaymentFormServiceMock, getStoreConfig } from '@bigcommerce/checkout/test-mocks';
+import { act } from '@bigcommerce/checkout/test-utils';
+
+import { getBigCommercePaymentsMethod } from '../mocks/paymentMethods.mock';
+import BigCommercePaymentsPaymentMethod from './BigCommercePaymentsPaymentMethod';
+
+describe('BigCommercePaymentsPaymentMethod', () => {
```

Comment:
```suggestion
describe('BigCommercePaymentsPayPalPaymentMethod', () => {
```

Also please update file name

Reply:
fixed